Makefile

```diff
@@ -36,6 +38,17 @@ binary: generate-webui build ## build the linux binary
 crossbinary: generate-webui build ## cross build the non-linux binaries
 	$(DOCKER_RUN_TRAEFIK) ./script/make.sh generate crossbinary
 
+crossbinary-parallel:
```

Comment:
Can't we just make this target depend on `binary` and remove the first two make calls?